### PR TITLE
Update readme 198

### PR DESCRIPTION
**Pull request by russelldutton**
Prettified REAMDE.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,95 @@
...
 [![Dependencies Status](https://david-dm.org/301-Redirection/301-Project/status.svg)](https://david-dm.org/301-Redirection/301-Project)
 [![DevDependencies Status](https://david-dm.org/301-Redirection/301-Project/dev-status.svg)](https://david-dm.org/301-Redirection/301-Project?type=dev)
-## Development server
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+The Dota 2 Bot Configurator is a tool that allows a Dota 2 player to specify a set of configuration options by means of a Web-based, Graphical User Interface from which a set of Lua scripts are generated. These scripts can be used by the game to alter the default bot behaviour in-game.
+
+#### Features
+
+- Exactly zero coding knowledge required. Write a bot scripts in minutes!
```
Comment (vector67) [inline README.md]: scripts should be script

File: README.md [hunk 2]
```diff
@@ -1,31 +1,95 @@
...
+- Save and edit multiple sets of bot scripts.
+
+
+
+## Running Locally
+
+#### Requirements
+
+- Git
+- Node.js (version 8 or 10)
```
Comment (vector67) [inline README.md]: Do we know it works with version 8?

File: README.md [hunk 3]
```diff
@@ -1,31 +1,95 @@
...
+
+Clone this repository with `git clone git@github.com:301-Redirection/301-Project.git`. 
+
+Make sure to run `npm install && npm run scrape` to install project dependencies and fetch the images for the heroes and items. 
+
+Run `npm start` to start. This will start the node server, running on http://localhost:3000 and the live server at http://localhost:4200. The app will automatically reload if you change any of the source files.
```
Comment (vector67) [inline README.md]: This is only true for the frontend

File: README.md [hunk 4]
```diff
@@ -1,31 +1,95 @@
...
+    </li>
+</ul>
 
-## Code scaffolding
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
 ## Build
 
 Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
```
Comment (vector67) [inline README.md]: Is it true that the stuff is stored in dist/

File: README.md [hunk 5]
```diff
@@ -1,31 +1,95 @@
...
 
 ## Running unit tests
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+Run `npm run test-frontend` to execute the unit tests via [Karma](https://karma-runner.github.io), `npm run test-backend` to execute the tests for the node server and `npm run test-all` to run both sets of tests.
```
Comment (vector67) [inline README.md]: This also runs the linters

File: README.md (hunk 2 above)
Comment (SpellBreakerZA) [reply README.md]: Yes, 8, 9 and 10

**Pull request merged**